src/startup/libheader.src

```diff
@@ -40,6 +40,7 @@ __notfound:
 
 	segment	libs
 __relocationstart:		; relocation data will be placed here
+	db $C0, "LibLoad", 0, 30 ; hardcode dependency on LibLoad v3.0, FIXME HACK
```

Comment:
Any ideas for how to add this dependency in a more proper manner? At least, one that pulls the appropriate version number from libload's source?

Reply:
A fasmg macro to include a file in a virtual space and then pull the symbol or something would do the trick

Reply:
Is there any good way to resolve this now?